dynamicannotationdb/annotation.py

```diff
@@ -3,26 +3,22 @@
 from typing import List
 
 from marshmallow import INCLUDE
-from sqlalchemy.orm.exc import NoResultFound
+from sqlalchemy import DDL, event
 
-from dynamicannotationdb.errors import (
+from .database import DynamicAnnotationDB
+from .errors import (
     AnnotationInsertLimitExceeded,
     NoAnnotationsFoundWithID,
     UpdateAnnotationError,
 )
-from dynamicannotationdb.interface import DynamicAnnotationInterface
-from dynamicannotationdb.models import AnnoMetadata
+from .models import AnnoMetadata
+from .schema import DynamicSchemaClient
 
 
-class DynamicAnnotationClient(DynamicAnnotationInterface):
-    def __init__(self, aligned_volume, sql_base_uri):
-        sql_uri = self.create_or_select_database(aligned_volume, sql_base_uri)
-        super().__init__(sql_uri)
-
-        self.aligned_volume = aligned_volume
-
+class DynamicAnnotationClient(DynamicAnnotationDB, DynamicSchemaClient):
+    def __init__(self, url: str = None, aligned_volume: str = None) -> None:
```

Comment:
is this interface change necessary?

Reply:
Prob not. It would be better to use a composition model but I didn't have time to change that. I don't like having inherited classes as it muddies the code

Reply:
its probably not too hard to change in Ann engine and Mat engine so its not a big deal